packages/sshnoports/bin/sshnp.dart

```diff
@@ -55,7 +54,10 @@ void main(List<String> args) async {
     }
 
     await sshnp.init();
-    await sshnp.run();
+    SSHNPResult res = await sshnp.run();
+    if (res is SSHCommand) {
+      stdout.write('$res\n');
+    }
     exit(0);
```

Comment:
Don't we need a test for `res is SSHNPFailed` which will result in an`exit(1);`?

Reply:
Good catch, thanks